src/meta/src/rpc/server.rs

```diff
@@ -466,3 +473,54 @@ pub async fn rpc_serve_with_store<S: MetaStore>(
 
     Ok((join_handle, shutdown_send))
 }
+
+#[cfg(test)]
+mod tests {
+    use std::thread::sleep;
+
+    use super::*;
+
+    #[tokio::test(flavor = "multi_thread")]
```

Comment:
Better not to have sleep in unit tests :) Ignoring this test looks like a better choice.

Reply:
Also tokio::sleep instead of thread::sleep?

Reply:
good idea.